integration/shareddata/composites.go

```diff
@@ -21,11 +21,13 @@ import "go.mongodb.org/mongo-driver/bson"
 // This shared data set is not frozen yet, but please add to it only if it is really shared.
 var Composites = &Values[string]{
 	data: map[string]any{
-		"document":       bson.D{{"foo", int32(42)}},
-		"document-empty": bson.D{},
+		"document":            bson.D{{"foo", int32(42)}},
+		"document-resembling": bson.D{{"foo", int32(42)}, {"42", "foo"}, {"array", bson.A{"foo", int32(42)}}},
```

Comment:
what means resembling? what to resemble? can we please specify more details?

Reply:
"document-resembling" is resembling the existed array and document above. I've created this for checking for equivalent the exact document or for exact array. But I just noticed that this document doesn't exactly match the existing array. I'll redo it. Do you have any suggestions or something to add or maybe it should to rename the document?

Reply:
I would suggest `composite` word.

Reply:
`document-composite`?

Reply:
yep